code/src/components/AddTodo.js

```diff
@@ -0,0 +1,64 @@
+import React from 'react';
```

Comment:
can shorten the code and write `import React, { useState } from 'react'` because they are from the same dependency.